### PR TITLE
Add 16 more portals from around Shoushi

### PR DESCRIPTION
**Pull request by Jyard1**

### CODE REVIEW

File: Database/Updates/World/010_2017_04_24_destination_portal.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+INSERT INTO `portal_destination` (`weenieClassId`, `cell`, `x`, `y`, `z`, `qx`, `qy`, `qz`, `qw`, `min_lvl`, `max_lvl`) VALUES
+(1124, 32965104, 40, -140, 0.005, 1, 0, 0, 0, 0, 0),
+(1096, 3662938154, 135.1000061, 42.70000076, 34, -0.96592581, 0, 0, -0.25881913, 0, 0),
```
Comment (Mogwai-AC) [inline Database/Updates/World/010_2017_04_24_destination_portal.sql]: I think you have your qx and qz values backwards, here.  QX rotation is like doing barrel-roll rotations.  No bueno.

**Comment by Mogwai-AC** [comment]
Also, you need to put a comment in the changelog to reflect what you did.

File: Database/Updates/World/010_2017_04_24_destination_portal.sql (hunk 1 above)
Comment (Jyard1) [inline Database/Updates/World/010_2017_04_24_destination_portal.sql]: Ah didn't see that /teleloc used a different order, will fix tanks.

**Comment by Jyard1** [comment]
Not sure what AppVeyor's problem is with the update to the changelog.

**Pull request closed without merging**